shell/src/main/java/tachyon/shell/TFsShell.java

```diff
@@ -113,6 +113,53 @@ public int cat(TachyonURI path) throws IOException {
   }
 
   /**
+   * Copies a file or directory specified by argv from the under filesystem to the filesystem. 
+   *
+   * @param argv [] Array of arguments given by the user's input from the terminal
+   * @return 0 if command is successful, -1 if an error occurred.
+   * @throws IOException
+   */ 
+  public int loadFile(TachyonURI filePath) throws IOException {
+    TachyonFS tachyonClient = createFS(filePath);
+    int ret = loadPath(tachyonClient, filePath);
+    if (ret == 0) {
+      System.out.println(filePath + " loaded");
+    } else {
+      System.out.println("Loading " + filePath + " failed");
+    }
+    return ret;
+  }
+
+  private int loadPath(TachyonFS tachyonClient, TachyonURI filePath) throws IOException {
+    TachyonFile tFile = tachyonClient.getFile(filePath);
+    Closer closer = Closer.create();
+    if (tFile == null) {
+      return -1;
+    }
+    if (!tFile.isDirectory()) {
```

Comment:
NIT: Since you are processing both branch cases any way, why not switch the two branches and make condition on `tFile.isDirectory()` rather than `!tFile.isDirectory()`